factory/bootstrap/bootstrapComponents.go

```diff
@@ -180,6 +189,7 @@ func (bcf *bootstrapComponentsFactory) Create() (*bootstrapComponents, error) {
 		HeaderIntegrityVerifier:    headerIntegrityVerifier,
 		DataSyncerCreator:          dataSyncerFactory,
 		ScheduledSCRsStorer:        nil, // will be updated after sync from network
+		TrieSyncStatisticsProvider: bcf.statusCoreComponents.TrieSyncStatistics(),
```

Comment:
I think we should increment by one the values for the processed nodes just before we start the heartbeat sender during bootstrap.
Reason: when the sender is instantiated, the very first heartbeat message should contain a non-0 value on the new `NumTrieNodesSynced` field so it will immediately be marked correctly in the explorer